pkg/lib/oauth/handler/handler_token.go

```diff
@@ -630,13 +630,30 @@ func (h *TokenHandler) handleRefreshToken(
 	client *config.OAuthClientConfig,
 	r protocol.TokenRequest,
 ) (protocol.TokenResponse, error) {
+	logger := TokenHandlerLogger.GetLogger(ctx)
 	deviceInfo, err := r.DeviceInfo()
 	if err != nil {
+		logger.WithSkipLogging().Error(ctx,
+			"failed to get device info from token request",
+			slog.String("error", err.Error()),
```

Comment:
I recommend we add an extra attribute `slog.Bool("refresh_token_log", true)` for easier searching.

Please add this attribute to previous log calls as well. So when we search this attribute, we can find all logs.

Reply:
@tung2744 I meant we also add this attribute to the log calls added in https://github.com/authgear/authgear-server/pull/5348/files

Reply:
I see, ok